guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -12,7 +12,7 @@ include::snip_firewalld.adoc[]
 ----
 # yum install rh-postgresql12-postgresql-server \
 rh-postgresql12-syspaths \
-rh-postgresql12-postgresql-evr
+rh-postgresql12-postgresql-server
```

Comment:
This doesn't make sense to me. This package is already listed on line 13 and the evr extension is really needed. On an all-in-one installation the installer can take care of it, but on an external server you need to do it manually.

Reply:
This contradicts what Kenny says in [BZ#1898111](https://bugzilla.redhat.com/show_bug.cgi?id=1898111)

Reply:
And I think that BZ is invalid. I've left a comment there.

Reply:
My mistake seems that evr contains the server as a dependency.

# rpm -qpR rh-postgresql12-postgresql-evr-0.0.2-1.el7sat.x86_64.rpm 
~~~
rh-postgresql12-postgresql-server
rh-postgresql12-runtime
rh-postgresql12-runtime
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
~~~